app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -88,7 +88,7 @@
             new_tab: true,
           },
           decorated_session.sp_name && {
-            url: idv_doc_auth_return_to_sp_path,
+            url: idv_doc_auth_return_to_sp_path(location: 'welcome'),
```

Comment:
Since this is a locator of sorts for a link within the step, maybe a name like "missing_items" could be a bit more precise?

```suggestion
            url: idv_doc_auth_return_to_sp_path(location: 'missing_items'),
```

Reply:
That makes sense - thanks!